solutions/devsprint-bruno-ramos-3/DeliveryApp/Screens/AddressSearch/AddressCellView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  AddressCellView.swift
+//  DeliveryApp
+//
+//  Created by Pedro Henrique Léda on 15/02/22.
+//
+
+import Foundation
+import UIKit
+
+class AddressCellView: UITableViewCell {
+    var stackView: UIStackView = {
+        let stack = UIStackView(frame: .zero)
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.spacing = 8
+        return stack
+    }()
+    
+    lazy var titleLabel: UILabel = {
+       let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 17)
+        label.text = "Rua dos Patinhos, 126"
+        return label
+    }()
+    
+    lazy var subtitleLabel: UILabel = {
+       let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.text = "Limoeiro"
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        addSubviews()
+        configureConstraints()
```

Comment:
Utilizando o protocolo `ViewCode` só sera necessário chamar o método `setup`
```suggestion
        setup()
```